www/aolserver/Portfile

```
@@ -1,49 +1,54 @@
-PortSystem 1.0
-
-name             aolserver
-version          4.5.1
-categories       www
-maintainers      {toby @tobypeterson}
-description      America Online's open-source web server
-long_description \
-    AOLserver is America Online's open-source web server. AOLserver is the \
-    backbone of the largest and busiest production environments in the \
-    world. AOLserver is a multithreaded, Tcl-enabled web server used for \
-    large scale, dynamic web sites.
-homepage         https://aolserver.github.io/
-platforms        darwin
-license          MPL-1.1-AOLserver
-
-master_sites     sourceforge
-distname         ${name}-${version}-src
-checksums        md5    d7aa89d6b864d75460fafd2f26553fec \
-                 sha1   9b37a883b34902d80bf042a1bd8d274f585fe843 \
-                 rmd160 c35faa53ecae93e494f4c44dc1bf3594391562ce
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        aolserver aolserver db30f6c739824283d543fedd7e358e9335a45745
```

The changes in patch-panic.diff have since been merged upstream; consider using the latest commit instead:

```suggestion
github.setup        aolserver aolserver 650ff5697c8b423f7cc338fbca64ac7f1d891e99
```